blueocean-dashboard/src/main/js/components/RunDetailsTests.jsx

```diff
@@ -81,9 +78,9 @@ RunDetailsTests.propTypes = {
     params: PropTypes.object,
     isMultiBranch: PropTypes.bool,
     result: PropTypes.object,
-    testResults: PropTypes.object,
-    resetTestDetails: PropTypes.func,
-    fetchTestResults: PropTypes.func,
+    tests: PropTypes.object,
```

Comment:
any reason for this name change? makes this a much bigger diff than it may other wise be